diff -Naur processes.orig processes
--- processes.orig      2014-06-29 05:53:28.000000000 +0200
+++ processes   2014-06-29 05:54:35.000000000 +0200
@@ -71,10 +71,10 @@
 if [ "$1" = "autoconf" ]; then
         case "$OPERSYS" in
         Linux|SunOS|FreeBSD|OpenBSD|NetBSD|Darwin|CYGWIN)
-        $ps >/dev/null 2>/dev/null
+        $ps -A >/dev/null 2>/dev/null
         if [ $? -ne 0 ]
         then
-            echo "no (ps=$ps failed)"
+            echo "no (ps=$ps -A failed)"
             exit 0
         fi
         echo | $awk '{ print "Hei" }' >/dev/null 2>/dev/null